robottelo/cli/user.py

```diff
@@ -107,6 +107,16 @@ def access_token(cls, action=None, options=None):
         cls.command_sub = f'access-token {action}'
         return cls.execute(cls._construct_command(options), output_format='csv')
 
+    @classmethod
+    def registration_tokens(cls, options=None):
+        """
+        Usage:
+        hammer user registration-tokens [OPTIONS]
+
+        """
+        cls.command_sub = 'registration-tokens invalidate'
```

Comment:
@shweta83 what about the `registration-tokens invalidate-multiple` sub command? Don't we want to cover that too?

Reply:
Updated both. Please check.